README.md

```diff
@@ -104,6 +108,47 @@ at runtime as an environment variable:
 
 See `giftless/config.py` for some default configuration options. 
 
+
+### Running a local example
+
+First, create a new project on Github or any other platform.
+Here, we create `example-proj-datahub-io`. Add any data file
+to it. The goal is to track this possible large file with 
+git-lfs and use Giftless as the local server. In our example,
+we create a CSV named `research_data_factors.csv`.
+
+Next, start the Giftless server. Go to your local project 
+directory and run:
+
+```bash
+example-proj-datahub-io$ git init
+example-proj-datahub-io$ git remote add origin YOUR_REMOTE_REPO
+example-proj-datahub-io$ git lfs track '*.csv'
+example-proj-datahub-io$ git lfs track 
```

Comment:
Is it really necessary to have two `git lfs track` commands?

Reply:
The second one lists the tracked files. I can add a comment clarifying this

Reply:
@hannelita i would not track *.csv but rather track a specific csv files atm ...